kebechet/managers/version/version.py

```diff
@@ -255,9 +263,24 @@ def _compute_changelog(
             # can be in case of the very first release.
             old_version = repo.git.rev_list("HEAD", max_parents=0)
 
-        changelog = repo.git.log(
-            f"{old_version}..HEAD", no_merges=True, format="* %s"
-        ).splitlines()
+        _LOGGER.info("Smart Log : " + str(changelog_smart))
+
+        if changelog_smart:
+            _LOGGER.info("Classifier : " + changelog_classifier)
+            _LOGGER.info("Format : " + changelog_format)
+            changelog = repo.git.log(
+                f"{old_version}..HEAD", no_merges=True, format="%s"
+            ).splitlines()
+            changelog = generate_log(
```

Comment:
If you want to be precise, you can catch any exception from the thoth-glyph library (the top level exception from the library's exception hierarchy) and report to user (via closing issue comment) that there is something wrong in the config (with the exception text).

Reply:
Catching Thoth's exceptions now, and closing the issue with a comment (with the exception text).